helper/schema/grpc_provider.go

```diff
@@ -21,7 +22,15 @@ import (
 	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
 )
 
-const newExtraKey = "_new_extra_shim"
+const (
+	newExtraKey        = "_new_extra_shim"
+	tflogSubsystemName = "sdkv2"
```

Comment:
I'm not the biggest fan of including the version number in the logging key as it could be missed during upgrades, but understandably the ability to differentiate between subsystems is important.

Reply:
What do we think of `helper_schema`?

Reply:
Love it, great suggestion.